src/services/chris_integration.tsx

```diff
@@ -86,13 +87,26 @@ export const modifyDatetime = (oldDay: string): string => {
   return rvtVal
 }
 
-class ChrisIntegration {
+export const isNotModel = (modelName: string): boolean => { // Dynamically loop through all model plug-ins and check if the current plug-in is valid
+  let xrayKey: keyof typeof app.XrayModels;
+  let ctKey: keyof typeof app.CTModels;
+  
+  for (xrayKey in app.XrayModels) {
+    if (modelName === app.XrayModels[xrayKey]) {
+      return false;
+    }
+  }
+
+  for (ctKey in app.CTModels) {
+    if (modelName === app.CTModels[ctKey]) {
+      return false;
+    }
+  }
```

Comment:
Have you considered using `.forEach` or `Object.values(dict).includes(value)`? Might look a bit cleaner and you will also be able to avoid defining union of literals type. @janakitti what do you think?

Reply:
I think I'd go with the second suggestion since it allows you to make it a one-liner. Although, I think you meant to write: `Object.keys(dict).includes(value)`

```
export const isNotModel = (modelName: string): boolean =>
!(Object.keys(app.XrayModels).includes(modelName) || Object.keys(appCTrayModels).includes(modelName))
```

Reply:
Yes, the suggestion you wrote down is what I had in mind.

I might be wrong but I think @colinywu is checking for values and not keys.